### PR TITLE
fix: Filters in explore overflow 

### LINKED ISSUE #8762: Filters in explore overflow
image
- the item count should not jump to another row
- the filters should 
- expected: in this case whirls and clear all are on the second row with sufficient top margin - something like 12px (between the rows) 

image

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #8762

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2023-12-26 at 16 31 33@2x

## Copilot Summary
copilot:summary

copilot:poem

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]

**Review by daiagi — APPROVED** [review]

**Comment by exezbcz** [comment]
working nicely, one thing is the item count 

image


could you please center it out

thanks!

File: components/items/Items.vue [hunk 1]
```diff
@@ -5,10 +5,12 @@
       <div class="is-hidden-mobile">
         <div class="flex justify-between pb-4 pt-5 content-center">
           <BreadcrumbsFilter />
-          <div v-if="isLoading">
-            <NeoSkeleton no-margin :width="80" />
+          <div class="mt-1 shrink-0">
```
Comment (roiLeo) [inline components/items/Items.vue:8]: or `align-items: center;` on wrapper div instead of adding margin-top

Comment (roiLeo) [reply components/items/Items.vue:8]: wdyt @exezbcz

**Comment by yangwao** [comment]
Merging for release, you can do follow up @roiLeo 

thanks!
pay 20 usd

**Pull request merged**